### PR TITLE
submission of homework week1

### PR DESCRIPTION
**Pull request by hnnhack**

### CODE REVIEW

File: Week1/homework/queries.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  Population INT DEFAULT 0,
+  LifeExpectancy DECIMAL(3,1),
+  GNP FLOAT(10,2),
+  GNPOld FLOAT(10,2),
+  LocalName VARCHAR(45) NOT NULL,
+  GovernmentForm VARCHAR(45),
+  HeadOfState VARCHAR(60),
+  Capital VARCHAR(50) NOT NULL,
+  PRIMARY KEY (Code));
+`,
```
Comment (mogaleaf) [inline Week1/homework/queries.js]: Yes this Insert is good .

File: Week1/homework/queries.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  `CREATE TABLE city (
+  ID INT(11) NOT NULL AUTO_INCREMENT,
+  Name CHAR(35) NOT NULL DEFAULT '',
+  CountryCode CHAR(3) NOT NULL DEFAULT '',
+  District CHAR(20) NOT NULL DEFAULT '',
+  Population INT(11) NOT NULL DEFAULT '0',
+  PRIMARY KEY (ID),
+  KEY CountryCode (CountryCode),
+  CONSTRAINT city_ibfk_1 FOREIGN KEY (CountryCode) REFERENCES country (Code)
+);
```
Comment (mogaleaf) [inline Week1/homework/queries.js]: Yes good and for the foreign key too

File: Week1/homework/queries.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  GNPOld FLOAT(10,2),
+  LocalName VARCHAR(45) NOT NULL,
+  GovernmentForm VARCHAR(45),
+  HeadOfState VARCHAR(60),
+  Capital VARCHAR(50) NOT NULL,
+  PRIMARY KEY (Code));
+`,
+  // QUERIES[4] = command 5: inserts data into table country;
+  `INSERT INTO country 
+  (Code, Name, Continent, Region, SurfaceArea, IndepYear, Population, LifeExpectancy, GNP, GNPOld, LocalName, GovernmentForm, HeadOfState, Capital)
```
Comment (mogaleaf) [inline Week1/homework/queries.js]: good insert

File: Week1/homework/queries.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  Population INT(11) NOT NULL DEFAULT '0',
+  PRIMARY KEY (ID),
+  KEY CountryCode (CountryCode),
+  CONSTRAINT city_ibfk_1 FOREIGN KEY (CountryCode) REFERENCES country (Code)
+);
+`,
+  // QUERIES[6] = command 7: inserts data into table city;
+  `INSERT INTO city 
+  (Name, CountryCode, District, Population)
+VALUES
```
Comment (mogaleaf) [inline Week1/homework/queries.js]: Good too, you are using the auto increment

File: Week1/homework/assignment.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    "query": "SELECT name FROM country WHERE Continent = 'Europe';"
+  },
+  {
+    "name": "5. List all the countries in the descending order based on their surface areas.",
+    "query": "SELECT name FROM country ORDER BY SurfaceArea DESC;"
+  },
+  {
+    "name": "6. What are the names of all the cities in the Netherlands?",
+    "query": "SELECT city.name AS CityName FROM city INNER JOIN country ON city.CountryCode = country.Code WHERE country.name = 'Netherlands';"
+  },
```
Comment (mogaleaf) [inline Week1/homework/assignment.json]: It's a good Join , Inner is not necessary keyword as it's inner by default

Comment (hnnhack) [reply Week1/homework/assignment.json]: Yes, you are right,..
Thank you very much for review,..

**Pull request closed without merging**